docs/tutorial/1-connect-to-asset-hub.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+slug: /connect-to-asset-hub
+---
+
+import {WestendNetworkButton, LocalNetworkButton} from '@site/src/components/NetworkButton';
+
+# Connect to Asset Hub
+
+Install the [MetaMask] browser extension, and click on the button below to connect to a network.
+Alternatively, you can [manually add a custom network][add-network] using the settings provided below.
+
+# Connect to Asset Hub Westend Testnet
+
+Connect your Metamask wallet to Asset-Hub using the following link:
+
+<WestendNetworkButton />
+
+<details>
+<summary>Or add it manually</summary>
+- Network name: Asset-Hub Westend Testnet
+- RPC URL URL: `https://westend-asset-hub-eth-rpc.polkadot.io`
+- Chain ID: `420420421`
+- Currency Symbol: `WND`
+- Block Explorer URL: `https://assethub-westend.subscan.io`
+</details>
+
+# Local development
+
+Connect your Metamask wallet to a local server using the following link:
+
+<LocalNetworkButton />
```

Comment:
I believe this should be a separate tutorial, such as Working in a Local Environment. Including it here doesn’t seem necessary, as most users aren’t interested in setting up a local environment. This content would be more relevant if directed toward advanced users only.